app/elements/socobo-inventory/socobo-inventory-create.html

```diff
@@ -199,7 +204,11 @@
             this.item = item;
             this.$.bestBeforeInput.value = item.bestBefore;
             this.$.categoryDropdownMenu.select(this.categories.indexOf(item.category));
-            this.$.measurementDropdownMenu.select(this.measurements.indexOf(item.measurement));
+            this.measurements.forEach(function(measurement, index) {
+              if (this.measurements.indexOf(item.measurement) !== -1) {
```

Comment:
Yeah, this doesn't work with `Object`. You can see that if you play around with the different dialogs. The dropdown will remember the last thing you selected, i.e. it is not reset properly.